apps/server/src/services/users/users.controller.ts

```diff
@@ -0,0 +1,19 @@
+import { Body, Controller, Get, Post } from '@nestjs/common';
+import { UsersService } from './users.service';
+import { User } from './user.model';
+import { UserDto } from './dto/user.dto';
+
+@Controller('users')
+export class UsersController {
+  constructor(private readonly usersService: UsersService) {}
+
+  @Get('/getUsers/')
```

Comment:
no need for '/getUsers/'. why to create a new route instead of the one that you already have?